README.md

```diff
@@ -43,6 +43,11 @@ You can find a great example of such an issue [here](https://github.com/nodejs/c
 
 Community Committee meetings will be broadcast via Zoom, will be announced ahead of time for access, and coordinated to optimize for contributor timezones.
 
+Meeting cadence is every other week on Thursdays.  Please visit the [issues](https://github.com/nodejs/community-committee/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+%22Node.js+Foundation+Community+Committee+Meeting%22) section for next scheduled meeting.  If none found, please feel free to ping any of the [members](#community-committee-members).
```

Comment:
> If none found, please feel free to ping any of the [members](#community-committee-members)

There generally won't be an issue open in the week between meetings. I'd suggest revising this to help reduce the maintenance burden on the CommComm members.

Reply:
I can change the verbiage to allude to newcomers going into the meetings folder and checking the most recent and work their way up to see when next one's scheduled.  

Quick question, however, how do CommComm members keep track and notified as to when the next meeting will be?  Just remember (bit risky)?  Notifications from the [Node calendar](https://nodejs.org/calendar)?  If so, any way for the public to subscribe to notifications?

Reply:
I would suggest pointing them to the nodejs google calendar instead which will show the meeting time/dates into the future.  We can also then mention that an issue will be created sometime before the meeting with the meeting specifics and to look for it in the commcomm repo.

Reply:
Will do!  Thx, @mhdawson ! 👊🏼